### PR TITLE
added new service ontology_api and edited environment of grisera_api

### PR DESCRIPTION
**Pull request by blazejfury**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -24,3 +24,11 @@ services:
...
       - GRAPH_API_HOST=host.docker.internal
       - GRAPH_API_PORT=18080
+      - ONTOLOGY_API_HOST=host.docker.internal
+      - ONTOLOGY_API_PORT=18082
+  ontology_api:
+    build: ontology_api
+    ports:
+      - "18082:80"
+    extra_hosts: 
+      - "host.docker.internal:host-gateway"
```
Comment (mrwrob) [inline docker-compose.yml:34]: For the purpose of the testing, we need another container of the `GriseraAPI`, that will run with the `OntologyAPI`. Within this container, there should be passed some variable, lets call it `PERSISTENCE_TYPE`, which value for OntologyAPI will be set to `2`. Please add:
```
  grisera_api_ontology:
    build: grisera_api_ontology
    ports:
      - "18083:80"
    extra_hosts: 
      - "host.docker.internal:host-gateway"
    environment:
      - ONTOLOGY_API_HOST=host.docker.internal
      - ONTOLOGY_API_PORT=18082
      - PERSISTENCE_TYPE=2
```

Comment (mrwrob) [reply docker-compose.yml:34]: To be clear, now we will have 5 containers:
- `grisera_neo4j` - Neo4J graph database
- `graph_api` - backend bridge between `grisera_api` and graph database
- `ontology_api`- backend that allows to process ontology file
- `grisera_api` - REST API working with the graph database
- `grisera_api_ontology` - REST API working with the ontology backend

Both `grisera_api` and `grisera_api_ontology` will be publicly available.

**Review by mrwrob — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (blazejfury) [reply docker-compose.yml:34]: gotowe

**Review by mrwrob — APPROVED** [review]
Good job :+1:

**Review by KKucht — APPROVED** [review]
It looks fine for me.

File: docker-compose.yml (hunk 1 above)
Comment (borsuczek) [reply docker-compose.yml:47]: OK.

**Review by borsuczek — APPROVED** [review]

**Pull request merged**